test/e2e/shared/aws_helpers.go

```diff
@@ -8,7 +8,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
are these changes intentional?

Reply:
These are the new files, added with GC PRs, linter was failing. Boilerplate was not matching with the existing ones.